.github/workflows/compile.yml

```diff
@@ -198,7 +200,12 @@ jobs:
       - name: Rearrange MacOS files
         if: ${{ github.event.inputs.macos }}
         run: |
-          cp artifacts/llama-bin-macos-arm64.dylib/libllama.dylib deps/libllama.dylib
+          mkdir deps/macos-cpu
+          cp artifacts/llama-bin-macos-cpu.dylib/libllama.dylib deps/macos-cpu/libllama-cpu.dylib
```

Comment:
I don't _think_ this naming will work.

As I understand it since we've marked our methods with `[DllImport("libllama")]` the file that gets loaded needs to be called `libllama`. That's why all of the other files are distinguished by their _folder_ path but have identical names (except for the file extension of course).

Reply:
If it's for GitHub artifacts, that could be fine. You would need to rename it for the Nuget since it would have to be the same as the DLLImport, but that could be done in a script whenever that gets generated.

Reply:
That's right it was the intention